css/mobile-screen.css

```diff
@@ -0,0 +1,353 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  margin: 0 25px;
+  width: 304px;
+  height: 103px;
+  color: rgb(251 128 39);
+}
+
+.title span {
+  color: rgb(251 128 39 / 70%);
+}
+
+.primary-text {
+  line-height: 150%;
+  font-size: 16px;
+  height: auto;
+  text-align: justify;
+  width: 80%;
+}
+
+.meeting-date-place {
+  padding: 10px 0;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.date {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  color: rgb(0 0 0 / 60%);
+}
+
+.place {
+  margin: 4px;
+}
+
+.main-program {
+  width: 100%;
+  height: 44em;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  background: url("../img/pattern_bg.png");
+  background-color: #272a32;
+}
+
+.main-program-title {
+  color: #fff;
+}
+
+.separator {
+  width: 53px;
+  height: 1em;
+  border-top: 1px solid rgb(251 128 39 / 100%);
+  padding: 0 0 0.5em;
+}
+
+.program-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.program-list-item {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width: 95%;
+  height: 4.6em;
+  padding: 3px 16px;
+  background: rgb(255 255 255 / 10%);
+  margin: 0.6em 0;
+}
+
+.program-list-item h3 {
+  display: flex;
+  align-items: center;
+  color: rgb(251 128 39 / 100%);
+  width: 5em;
+  height: 3.2em;
+}
+
+.program-list-item-img {
+  display: flex;
+  justify-content: center;
+  width: 4em;
+  height: 3.5em;
+  padding: 0.2em;
+  margin: 0.4em;
+}
+
+.program-list-item-img img {
+  width: 64px;
+  height: 56px;
+}
+
+.program-list-item p {
+  width: 13em;
+  height: 4em;
+  color: #fff;
+  font-size: 0.9em;
+}
+
+.program-btn {
+  width: 80%;
+  height: 4.3em;
+  background-color: rgb(251 128 39 / 100%);
+  color: #fff;
+  font-size: 1em;
+  margin-top: 0.9em;
+  transition: all ease-in-out 0.5s;
+}
+
+.see-program {
+  display: none;
+}
+
+.program-btn:hover {
+  background-color: rgb(251 128 39 / 50%);
+}
+
+.featured-speakers.active {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 60em;
+}
+
+.featured-speakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 33.4em;
+}
+
+.featured-speakers-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 6em;
+}
+
+.featured-speakers-list.active {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 68em;
+  padding: 0;
+  margin: 0;
+}
+
+.featured-speakers-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 33em;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
- [ ] Kindly make sure to align your speaker list cards correctly by leaving space on left side of each card in order to meet the provided design :wink: 

   - ### Your design
   
   ![image](https://user-images.githubusercontent.com/87197412/197244860-9559fa7f-278c-44e9-b5a2-a7346f279b8e.png)


   - ### Provided design
 
   ![image](https://user-images.githubusercontent.com/87197412/197244933-38768c0d-f311-4681-a1dd-34bf7221e1d9.png)